airflow/contrib/operators/hive_to_dynamodb.py

```diff
@@ -89,7 +90,8 @@ def execute(self, context):
         self.log.info('Extracting data from Hive')
         self.log.info(self.sql)
 
-        data = hive.get_pandas_df(self.sql, schema=self.schema)
+        data = hive.get_pandas_df(self.sql, schema=self.schema,
+                                  hive_conf=context_to_airflow_vars(context))
```

Comment:
This causes [this] test to fail (and [this](https://github.com/apache/incubator-airflow/blob/master/tests/contrib/operators/test_hive_to_dynamodb_operator.py#L98)).

[this]: https://github.com/apache/incubator-airflow/blob/master/tests/contrib/operators/test_hive_to_dynamodb_operator.py#L140